pr-data.csv

```diff
@@ -1555,7 +1555,21 @@ https://github.com/apache/dubbo,58d738682f02f0a315d72857b594eb9eace5e13d,dubbo-c
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.ReferenceConfigTest.testInjvm,OD-Vic,,,
 https://github.com/apache/dubbo,58d738682f02f0a315d72857b594eb9eace5e13d,dubbo-config/dubbo-config-api,org.apache.dubbo.config.ReferenceConfigTest.testLargeReferences,ID,Opened,https://github.com/apache/dubbo/pull/15766,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.ReferenceConfigTest.testReferenceRetry,NIO,DeveloperFixed,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.ServiceConfigTest.testProxy,NIO,Accepted,https://github.com/apache/dubbo/pull/7108,
```

Comment:
Please don't delete this old info, as it fixed a different cause of non-determinism; one test may have multiple causes.